components/PlantForm.js

```diff
@@ -215,23 +169,8 @@ const OptionsContainer = styled.div`
   flex-wrap: wrap;
 `;
 
-const RadioLabel = styled.label`
-  display: flex;
-  align-items: center;
-  gap: 5px;
-  font-size: 0.95rem;
-  color: #555;
-`;
-
-const CheckboxLabel = styled.label`
-  display: flex;
-  align-items: center;
-  gap: 5px;
-  font-size: 0.95rem;
-  color: #555;
-`;
-
-const StyledButton = styled.p`
+const StyledButton = styled.div`
```

Comment:
You might could think of another naming? Since it is a div but still you want to style your button maybe it is a StyledButtonContainer or something?